MapboxNavigationTests/Sources/Tests/NavigationViewControllerTests.swift

```diff
@@ -257,7 +260,7 @@ class NavigationViewControllerTestable: NavigationViewController {
                   dayStyle: Style,
                   styleLoaded: XCTestExpectation) {
         self.styleLoadedExpectation = styleLoaded
-        super.init(for: route, dayStyle: dayStyle, directions: Directions(accessToken: "abc", host: ""), voiceController: FakeVoiceController())
+        super.init(dayStyle: dayStyle, directions: Directions(accessToken: "abc", host: ""), voiceController: FakeVoiceController())
```

Comment:
Maybe it's not necessary for the tests to pass currently, but I think you might save someone surprise in the future if you set the route they've passed in as an arg.

Reply:
good catch, thanks